crawler.js

```diff
@@ -310,6 +317,16 @@ class Crawler {
     if (!argv.scope) {
       //argv.scope = url.href.slice(0, url.href.lastIndexOf("/") + 1);
       argv.scope = [new RegExp("^" + this.rxEscape(argv.url.slice(0, argv.url.lastIndexOf("/") + 1)))];
+      if (argv.urlFileList != "") {
+        let urlFileList = fs.readFileSync( path.join(__dirname, argv.urlFileList), "utf-8").split("\n");
+        for (let url of urlFileList) {
+          if (url.substr(-1) != "/"){
+            url = url + "/";
+          }
+          let urlScope = new RegExp("^" + this.rxEscape(url.slice(0, url.lastIndexOf("/") + 1)));
+          argv.scope.push(urlScope);
```

Comment:
I wonder if we should just add a catch all regex, `.*` instead of adding one for each URL?
Since we're only going through a fixed list, so number of regexes to check doesn't grow with the list.